tests/test_run_config.py

```diff
@@ -41,6 +41,21 @@
     num_cpus = 4
 
 
+def test_lazy_import(simulation_input):
```

Comment:
not really a lazy import. it's a lazy failure...
```suggestion
def test_import_utility(simulation_input):
```